code/modules/cargo/random_stock/t1_common.dm

```diff
@@ -0,0 +1,591 @@
+		/obj/item/weapon/storage/pill_bottle/dylovene, \
+		/obj/item/weapon/storage/pill_bottle/inaprovaline, \
+		/obj/item/weapon/storage/pill_bottle/kelotane, \
+		/obj/item/weapon/storage/pill_bottle/spaceacillin, \
+		/obj/item/weapon/storage/pill_bottle/tramadol \
+	)
+	var/newtype = pick(options)
+	new newtype(L)
+
+STOCK_ITEM_COMMON(cosmetic, 2.2)
+	if (prob(50))
+		new /obj/item/weapon/lipstick/random(L)
+	else
+		new /obj/item/weapon/haircomb(L)
+
+STOCK_ITEM_COMMON(suitcooler, 1.2)
+	new /obj/item/device/suit_cooling_unit(L)
+
+STOCK_ITEM_COMMON(officechair, 1.2)
+	var/turf/T = get_turf(L)
+	if (!turf_clear(T))
+		for (var/turf/U in range(T,1))
+			if (turf_clear(U))
+				T = U
+				break
+	new /obj/structure/bed/chair/office/dark(T)
+
+STOCK_ITEM_COMMON(booze, 3.7)
+	if (prob(8))//Spare keg of beer or xuizi juice
+		var/turf/T = get_turf(L)
+		if (!turf_clear(T))
+			for (var/turf/U in range(T,1))
+				if (turf_clear(U))
+					T = U
+					break
+
+		if (prob(80))
+			new /obj/structure/reagent_dispensers/beerkeg(T)
+		else
+			new /obj/structure/reagent_dispensers/xuizikeg(T)
+	else
+		var/list/drinks = typesof(/obj/item/weapon/reagent_containers/food/drinks/bottle)
+		drinks -= /obj/item/weapon/reagent_containers/food/drinks/bottle
+
+		for (var/i in 1 to rand(1, 3))
+			var/type = pick(drinks)
+			new type(L)
+
+STOCK_ITEM_COMMON(plant, 3.5)
+	var/turf/T = get_turf(L)
+	if (!turf_clear(T))
+		for (var/turf/U in range(T,1))
+			if (turf_clear(U))
+				T = U
+				break
+	new /obj/structure/flora/pottedplant/random(T)
+
+STOCK_ITEM_COMMON(bag, 3.5)
+	var/type = pick( \
+		/obj/item/weapon/storage/bag/trash, \
+		/obj/item/weapon/storage/bag/plasticbag, \
+		/obj/item/weapon/storage/bag/ore, \
+		/obj/item/weapon/storage/bag/plants, \
+		/obj/item/weapon/storage/bag/sheetsnatcher, \
+		/obj/item/weapon/storage/bag/cash, \
+		/obj/item/weapon/storage/bag/books \
+	)
+	new type(L)
+	if (prob(30))
+		new type(L)
+
+STOCK_ITEM_COMMON(extinguish, 2.2)
+	for (var/i in 1 to rand(1, 3))
+		var/type = pick( \
+			/obj/item/weapon/extinguisher, \
+			/obj/item/weapon/extinguisher/mini \
+		)
+		new type(L)
+
+STOCK_ITEM_COMMON(hailer, 1.1)
+	if (prob(50))
+		new /obj/item/device/megaphone(L)
+	else
+		new /obj/item/device/hailer(L)
+
+//A target, for target practice
+//Take em up to science for gun testing
+STOCK_ITEM_COMMON(target, 2)
+	var/turf/T = get_turf(L)
+	if (!turf_clear(T))
+		for (var/turf/U in range(T,1))
+			if (turf_clear(U))
+				T = U
+				break
+	if (prob(50))
+		new /obj/item/target(T)
+	else
+		new /obj/structure/target_stake(T)
+
+STOCK_ITEM_COMMON(snacks, 4)
+	//Snackboxes are much more likely to spawn on tables than in crates.
+	//This ensures the cargo bay will have a supply of food in an obtainable place for animals
+	//allows nymphs and mice to raid it for nutrients, and thus gives playermice more
+	//reason to infest the warehouse
+	if (CS && prob(65))
+		if (!isturf(L))
+			L = get_turf(pick(CS.tables))
```

Comment:
I think this is missing the item path? /obj/item/weapon/storage/box/snack

There also be a 50% change of being a /obj/item/weapon/storage/box/produce instead